src/exo5 - Nested data with traverse/exo5.ts

```diff
@@ -94,9 +94,12 @@ export const naiveGiveCurrencyOfCountryToUser = (
 // HINT: `option.traverse` asks for an Applicative as the first parameter. You
 // can find it for `Task` in `task.ApplicativePar`
 
-export const getCountryCurrencyOfOptionalCountryCode: (
+export const getCountryCurrencyOfOptionalCountryCode = (
   optionalCountryCode: Option<CountryCode>,
-) => Task<Option<Currency>> = unimplementedAsync;
+): Task<Option<Currency>>  => 
+  pipe (
+    option.traverse(task.ApplicativePar)(getCountryCurrency)(optionalCountryCode)
+  );
```

Comment:
Actually here:
```
export const method = (value: number): sthg => pipe(otherMethod(anotherMethod)(value))
```
Is equivalent to:
```
export const method = (value: number): sthg = otherMethod(anotherMethod)
```
And as you can see the 2nd one is way easier to read :)

Reply:
It is, thanks!